static/scss/partials/dashboard.scss

```diff
@@ -122,6 +115,12 @@
                 -o-transform:      rotate( 135deg );
                 -webkit-transform: rotate( 135deg );
```

Comment:
Sorry, I haven't looked at this codebase in a long time, but do we want autoprefixer for adding vendor prefixes?

Reply:
Nah – This smells like copy pasta. Fixing!